src/components/task/task.tsx

```diff
@@ -55,33 +55,32 @@ const TaskComponent = ({
             </div>
           </div>
           <button className="btn btn-primary btn-sm btn-block" onClick={() => completeTask(index)}>
-            {" "}
             <img src={check} className="image"></img> Task Complete
           </button>
-          <button className="btn btn-primary btn-sm btn-block" onClick={() => setIsOpen(true)}>
+          <button className="btn btn-primary btn-sm btn-block" onClick={() => setIsDeleteModalOpen(true)}>
             <img src={bin} className="image"></img> Delete Task
           </button>
         </div>
       </div>
-      {isOpen && (
-        <div className="modal show" tabIndex={-1} role="dialog">
-          <div className="modal-dialog" role="document">
+      {isDeleteModalOpen && (
+        <div className="modal fade show d-block" role="dialog">
```

Comment:
The default visibility for modals is `display: none;`, so you must override it. This is what I meant by "inspect why isn't it shown".

Reply:
My guess is that you can remove both fade and show. The answer for what changes the display is - inspect and you'll know.